File: Łatwe/Java/Wycinanie literek/WycinanieLiterek.java
import java.util.Scanner;

class WycinanieLiterek {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        while (in.hasNext()) {
            char c = in.next().charAt(0);
            String word = in.next();
            for (int i = 0; i < word.length(); i++) {
                if (word.charAt(i) != c)
                    System.out.print(word.charAt(i));
            }
            System.out.println();
        }
    }
}

File: Łatwe/C++/Wycinanie literek/WycinanieLiterek.cpp
#include <iostream>

int main() {
    char c;
    std::string word;
    while (std::cin >> c >> word) {
        for (int i = 0; i < word.length(); i++) {
            if (word[i] != c)
                std::cout << word[i];
        }
        std::cout << std::endl;
    }
}
